diff --git a/colors/solarized.vim b/colors/solarized.vim
index 70f5223..8976597 100644
--- a/colors/solarized.vim
+++ b/colors/solarized.vim
@@ -549,6 +549,8 @@ exe "hi! Constant"       .s:fmt_none   .s:fg_cyan   .s:bg_none
 "        Boolean         a boolean constant: TRUE, false
 "        Float           a floating point constant: 2.3e10
 
+exe "hi! Class"          .s:fmt_none   .s:fg_yellow .s:bg_none
+exe "hi! Member"         .s:fmt_bold   .s:fg_base0  .s:bg_none
 exe "hi! Identifier"     .s:fmt_none   .s:fg_blue   .s:bg_none
 "       *Identifier      any variable name
 "        Function        function name (also: methods for classes)
@@ -562,6 +564,7 @@ exe "hi! Statement"      .s:fmt_none   .s:fg_green  .s:bg_none
 "        Keyword         any other keyword
 "        Exception       try, catch, throw
 
+exe "hi! DefinedName"    .s:fmt_none   .s:fg_orange .s:bg_none
 exe "hi! PreProc"        .s:fmt_none   .s:fg_orange .s:bg_none
 "       *PreProc         generic Preprocessor
 "        Include         preprocessor #include
@@ -569,6 +572,8 @@ exe "hi! PreProc"        .s:fmt_none   .s:fg_orange .s:bg_none
 "        Macro           same as Define
 "        PreCondit       preprocessor #if, #else, #endif, etc.
 
+exe "hi! StorageClass"   .s:fmt_none   .s:fg_base01 .s:bg_none
+exe "hi! Structure"      .s:fmt_none   .s:fg_base01 .s:bg_none
 exe "hi! Type"           .s:fmt_none   .s:fg_yellow .s:bg_none
 "       *Type            int, long, char, etc.
 "        StorageClass    static, register, volatile, etc.
